torch/csrc/jit/passes/onnx/pattern_conversion/autograd_function_process.cpp

```diff
@@ -0,0 +1,62 @@
+#include <torch/csrc/jit/passes/onnx/pattern_conversion/autograd_function_process.h>
+
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+#include <torch/csrc/jit/passes/utils/subgraph_utils.h>
+
+namespace torch {
+namespace jit {
+
+void convertSubgraphToSubBlock(Block* block) {
```

Comment:
nit: wonder if we can create subblock directly for autograd node? Might affect other existing jit passes maybe?

Reply:
will create an issue to optimize this conversion

Reply:
please comment with link to issue if possible...